keyboards/kwstudio/scorpio_rev2/config.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2024 kwstudio
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define WS2812_PWM_DRIVER PWMD1
+#define WS2812_PWM_CHANNEL 3
+#define WS2812_PWM_COMPLEMENTARY_OUTPUT
```

Comment:
Datasheet indicates that B15 only supports TIM1_CH3N, so this would require the `WS2812_PWM_COMPLEMENTARY_OUTPUT` define to properly function.  Master still requires the pwm advnaced config (as I targeted develop for that patch).  

Since I suspect that this won't work at all, without this config, the PR should probably target develop, and have this define re-added.